### PR TITLE
Merge `Dockerfile`s in apache/skywalking-docker into this codebase

### LINKED ISSUE #7656: [Feature] Update SkyWalking Docker according to latest ElasticSearch client rebuild
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

After https://github.com/apache/skywalking/pull/7634 gets merged, we need to adjust docker files in https://github.com/apache/skywalking-docker for incoming 8.8.0 release.

### Use case

We don't need images for ElasticSearch 6 and 7 separate now.

### Related issues

https://github.com/apache/skywalking/issues/7471

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kezhenxu94**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #7656.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
let's try to chance jdk versions e2e testing? Remove some version or rename? It is strange they always fail on that mission only.

**Comment by kezhenxu94** [comment]
> Add a doc about how to run through docker and docker compose?
> docker repo should be archived later.

Added

File: .github/workflows/e2e.jdk-versions.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
     timeout-minutes: 90
     strategy:
       matrix:
-        jdk: [ 8, 9, 10, 11, 12, 13, 14, 15 ]
+        jdk: [ 8 ] # , 9, 10, 11, 12, 13, 14, 15 ] TODO: add it back
```
Comment (wu-sheng) [inline .github/workflows/e2e.jdk-versions.yaml:36]: This seems works. Let's try to skip 9 and 10 only?

File: docs/en/setup/backend/docker.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# How to use the Docker images
```
Comment (wu-sheng) [inline docs/en/setup/backend/docker.md:1]: I think we missed UI part and docker compose

File: docker/oap/Dockerfile.oap [hunk 3]
```diff
@@ -14,36 +14,113 @@
...
+	tar -xvf ${SKYWALKING_CLI_TGZ}; \
+    mkdir "bin/"; \
+	mv skywalking-cli-${SKYWALKING_CLI_VERSION}-bin/bin/swctl-${SKYWALKING_CLI_VERSION}-linux-amd64 bin/swctl; \
+	chmod 755 bin/*; \
+	command -v gpgconf && gpgconf --kill all || :; \
+	ls -la .;
 
 FROM $BASE_IMAGE
 
-RUN apk add --no-cache openssl || true
```
Comment (hanahmily) [inline docker/oap/Dockerfile.oap:23]: Could we remove "bash" here? `docker-entrypoint.sh` looks like not needing it anymore.

Comment (kezhenxu94) [reply docker/oap/Dockerfile.oap:23]: DONE

File: .github/workflows/e2e.jdk-versions.yaml (hunk 1 above)
Comment (kezhenxu94) [reply .github/workflows/e2e.jdk-versions.yaml]: Let's just drop 9, and 10

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**